br/backup-and-restore-overview.md

```diff
@@ -15,6 +15,34 @@ BR satisfies the following requirements:
 - Perform history data auditing to meet the requirements of judicial supervision.
 - Clone the production environment, which is convenient for troubleshooting, performance tuning, and simulation testing.
 
+## Before you use
+
+This section describes the prerequisites for using TiDB backup and restore, including the usage tips and compatibility issues.
+
+### Restrictions
+
+PITR:
+
+- PITR only supports restoring data to **an empty cluster**.
+- PITR only supports cluster-level restore and does not support database-level or table-level restore.
+- PITR does not support restoring data in user tables or privilege tables.
+
+### Some tips
+
+Snapshot backup:
+
+- It is recommended that you perform the backup operation during off-peak hours to minimize the impact on applications.
+- It is recommended that you execute multiple backup or restore operations one by one. Running backup operations in parallel leads to low performance. Worse still, lack of collaboration between multiple tasks might result in task failures and affect cluster performance.
+
+Snapshot restore:
+
+- BR uses resources of the target cluster as much as possible. Therefore, it is recommended that you restore data to a new cluster or an offline cluster. Avoid restoring data to a production cluster. Otherwise, your application might be affected.
```

Comment:
```suggestion
- BR uses resources of the target cluster as much as possible. Therefore, it is recommended that you restore data to a new cluster or an offline cluster. Avoid restoring data to a production cluster. Otherwise, your application will be affected inevitably.
```